src/containers/Header/Header.tsx

```diff
@@ -63,12 +63,7 @@ function Header({mainPage}: HeaderProps) {
         const breadcrumbs = getBreadcrumbs(page, options, rawBreadcrumbs, queryParams);
 
         return breadcrumbs.map((item) => {
-            const action = () => {
-                if (item.link) {
-                    history.push(item.link);
-                }
-            };
-            return {...item, action};
+            return {...item, href: item.link};
```

Comment:
It fully reloads page in this case. Can we make it `react-router` link, so the app will behave like SPA?